res/controllers/Denon-MC7000-scripts.js

```diff
@@ -724,36 +691,29 @@ MC7000.getPrevRateRange = function(currRateRange) {
     }
     return MC7000.negRateRanges[0];
 };
-        
+
 /* LEDs for VuMeter */
 // VuMeters only for Channel 1-4 / Master is on Hardware
 MC7000.VuMeter = function(value, group) {
-	var deckNumber = script.deckFromGroup(group),
-	    peak = 0x76, // where the red LED starts (clipping indicator)
-        level = value*value*value*value*0x69;
+    var VuMeterLEDPeakValue = 0x76,
+        VULevelOutValue = engine.getValue(group, "PeakIndicator") ? VuMeterLEDPeakValue : value*value*value*value*0x69,
+        deckNumber = script.deckFromGroup(group);
 
-    if (engine.getValue(group, "PeakIndicator")) {
-        var level = peak;
-    }
-    // now send the level meter signal to controller
-    midi.sendShortMsg(0xB0 + deckNumber - 1, 0x1F, level);
+    midi.sendShortMsg(0xB0 + deckNumber - 1, 0x1F, VULevelOutValue);
 };
 
 /* LEDs around Jog wheel */
 MC7000.JogLed = function(value, group) {
-	var deckNumber = script.deckFromGroup(group);
-	// do nothing before track starts
-	if (value < 0) return;
-	// While "VINYL" is active show spinny LEDs
-	if (MC7000.isVinylMode[deckNumber]) { 
-		var trackDuration = engine.getValue(group, "duration"),
- 	        position = value * trackDuration / 60 * MC7000.scratchParams.recordSpeed,
- 	        activeLED = Math.round(position * 96) % 96;
- 	// While "VINYL" is off show track position
- 	} else {
-  	    var activeLED = value * 96;
-  	};
-  	// sending the position of active LED to the controller
+    var deckNumber = script.deckFromGroup(group);
+    // do nothing before track starts
+    if (value < 0) return;
```

Comment:
Seems like an unnecessary simplification. Take a look at how I did it in my Numark NS6II mapping: 
https://github.com/mixxxdj/mixxx/pull/2049/files#diff-c0fbee09c5274a7da978d9c00a039f5bR628-R630 (Line 628-630)

Reply:
maybe - but this code is simple, does what it needs to do and I understand it. Thanks for the hint anyway

Reply:
Yes obviously. But possible other users would probably still like to have that feature. If you dont mind, I can try to implement it (should essentially be two lines more) and open a PR for you to test.

Reply:
Yep - very happy for any help. So glad you are looking into this in detail together with me. I just don't know how this would work because I also made local changes as I see your comments. As mentioned - first time on GitHub too so everything is new to me. Would need your advise. Thank you.

Reply:
I encourage you to dive into some git tutorials: https://github.com/dictcp/awesome-git#tutorial

https://learngitbranching.js.org/ is very good but maybe a bit abstract for your purposes.

Reply:
Thank you - I gonna look into this.